pkg/receiver/activedirectoryinvreceiver/README.md

```diff
@@ -0,0 +1,52 @@
+# Windows Active Directory inventory Receiver
+
+**Stability level**: Alpha
+
+A Windows Active Directory Inventory receiver collects inventory data from Active Directory. This includes information such as computer names, user names, email addresses, and location information
```

Comment:
Does this receiver only work on Windows? If so, the README should mention this. Also, the collector should fail fast if configured with this receiver on a non-Windows machine.

Reply:
yes only windows, I'll look into this one